components/alert/style/index.less

```diff
@@ -74,8 +74,15 @@
     overflow: hidden;
     font-size: @font-size-sm;
     line-height: 22px;
+    border: none;
+    background-color: transparent;
     cursor: pointer;
 
+    &:focus {
```

Comment:
Maybe it's better to remove `:focus`, it looks like before.
![test2](https://user-images.githubusercontent.com/24520193/64582143-3bfa8980-d3bf-11e9-8b98-efd9bfe141c9.gif)

Reply:
I agree with @orzyyyy .